bin/auth0setup/promptUser.js

```diff
@@ -0,0 +1,35 @@
+var fs = require('fs')
+var dotenv = require('dotenv')
+dotenv.config()
+var prompt = require('prompt');
+
+fs.writeFileSync('.env.local', '')
+
+prompt.start();
+
+prompt.get(['DOMAIN', 'EXPLORER_CLIENT_ID', 'EXPLORER_CLIENT_SECRET'], function (err, result) {
+  if (err) { return onErr(err); }
+
+  // overwrite .env file with access token
+  var envFile = fs.readFileSync('.env.local', 'utf8')
```

Comment:
😕 Here we're reading from a file that we are writing to on [line 6](https://github.com/codefordenver/members/pull/93/files#diff-f096c3631bef8aaea21378db9b0425c5R6) to be an empty file?

Reply:
Is this supposed to be reading from the `.sample-env` file?